src/v2/Components/ArtworkFilter/ArtworkFilters/FilterExpandable.tsx

```diff
@@ -1,11 +1,42 @@
-import { Expandable, ExpandableProps, useThemeConfig } from "@artsy/palette"
-import React from "react"
+import {
+  Box,
+  Expandable,
+  ExpandableProps,
+  useThemeConfig,
+} from "@artsy/palette"
+import React, { useContext, useEffect, useRef, useState } from "react"
+import { ScrollRefContext } from "./useScrollContext"
 
 export const FilterExpandable: React.FC<ExpandableProps> = props => {
   const tokens = useThemeConfig({
     v2: { mb: 1 },
     v3: { mb: 6 },
   })
 
-  return <Expandable mb={tokens.mb} {...props} />
+  // @ts-expect-error STRICT_NULL_CHECK
+  const { scrollRef } = useContext(ScrollRefContext)
+  const ref = useRef(null)
+
+  const [open, setOpen] = useState(!!props.expanded)
+
+  useEffect(() => {
+    if (open) {
+      // @ts-expect-error STRICT_NULL_CHECK
+      const height = ref.current.clientHeight
+
+      scrollRef.current.scrollTo({
+        top: height,
```

Comment:
That's not gonna work properly, I only made it scroll on click and show the height of expanded block. I have to figure out which equation is correct. Any hint would be highly appreciated

Reply:
Do you want its `top` via `getClientRect`?